server/src/test/java/com/google/coffeehouse/common/StorageHandlerHelperTest.java

```diff
@@ -0,0 +1,214 @@
+
+  public void insertBook() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Books")
+        .set("bookId")
+        .to("book")
+        .set("author")
+        .to("book")
+        .set("isbn")
+        .to("97800609459470")
+        .set("title")
+        .to("book")
+        .build());
+    dbClient.write(mutations);
+  }
+
+  @Test
+  public void checkPersonInClubQuery_personInClub() throws Exception {
+    insertPerson();
+    insertClub();
+    insertMembership();
+    Boolean actual = StorageHandlerHelper.checkPersonInClubQuery(dbClient, "person", "club");
+    Boolean expected = true;
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void checkPersonInClubQuery_personNotInClub() throws Exception {
+    insertClub();
+    Boolean actual = StorageHandlerHelper.checkPersonInClubQuery(dbClient, "personNotInClub", "club");
+    Boolean expected = false;
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void getMemberCountQuery_oneMembers() throws Exception {
+    insertPerson();
+    insertClub();
+    insertMembership();
+    long actual = StorageHandlerHelper.getMemberCountQuery(dbClient, "club");
+    long expected = 1;
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void getMemberCountQuery_noMember() throws Exception {
+    insertClub();
+    long actual = StorageHandlerHelper.getMemberCountQuery(dbClient, "club");
+    long expected = 0;
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void getPersonCountQuery_onePersonExists() throws Exception {
+    insertPerson();
+    long actual = StorageHandlerHelper.getPersonCountQuery(dbClient, "person");
+    long expected = 1;
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void getPersonCountQuery_noPersonExists() throws Exception {
+    long actual = StorageHandlerHelper.getPersonCountQuery(dbClient, "personThatDoesNotExist");
+    long expected = 0;
+    assertEquals(expected, actual);
+  }
+
+   @Test
+  public void getClubCountQuery_oneClubExists() throws Exception {
+    insertClub();
+    long actual = StorageHandlerHelper.getClubCountQuery(dbClient, "club");
+    long expected = 1;
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void getClubCountQuery_noClubExists() throws Exception {
+    long actual = StorageHandlerHelper.getClubCountQuery(dbClient, "clubThatDoesNotExist");
+    long expected = 0;
+    assertEquals(expected, actual);
+  }
+
+ @Test
+  public void getBookCountQuery_oneBookExists() throws Exception {
+    insertBook();
+    long actual = StorageHandlerHelper.getBookCountQuery(dbClient, "book");
+    long expected = 1;
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void getBookCountQuery_noBookExists() throws Exception {
+    insertPerson();
+    long actual = StorageHandlerHelper.getBookCountQuery(dbClient, "bookThatDoesNotExist");
+    long expected = 0;
+    assertEquals(expected, actual);
+  }
+}
```

Comment:
```suggestion
}

```
Please add newline at end of file.